### ISSUE TITLE
Suggestions! :D

### ISSUE THREAD

**Issue by MrNinja008** [reveenge/SkyBlockX#2]
My suggestion:
• pls remove inbulit scorehud and make scorehud addon for ifera scorehud plugin 6.0.0 less lag!
• pls add TopIsland Leaderbord support.
• Pls Add Custom Island Support inbulit and it will automatically adds to the ui we can able to set image for the custom island through config.
• pls add Island Size
• Pls add rename partners island to Co-operaters
• pls add Support for multiple islands Ex Friends Island that they can also join thier Friend island with perms

**Comment by reveenge** [comment]
I will add some of the features, but if you star, I would be very happy :) @MrNinja008

**Comment by reveenge** [comment]
I will not remove the scoreboard system because I wrote it myself

**Comment by reveenge** [comment]
I'm doing flying writing support to TopIsland, I'll share it soon

**Comment by reveenge** [comment]
I don't understand what you call Custom Island, I would appreciate it if you could explain it again.

**Comment by reveenge** [comment]
I am making the plugin with only 1 island, I will add various islands in the future.

**Comment by reveenge** [comment]
I will do the Welcome Message event shortly

**Comment by reveenge** [comment]
I will make the NPC system soon

**Comment by MrNinja008** [comment]
> I will not remove the scoreboard system because I wrote it myself

Oh okay but can you add option to disable that in config and make skyblock x addon for ifera Scorehud 6.0.0 so less lag tags: {island_size} {island_lock_status} it shows island locked or not {island_members}?

**Comment by MrNinja008** [comment]
> I don't understand what you call Custom Island, I would appreciate it if you could explain it again.

I mean we can able to make a custom island world and then we can do /is set pos1 and /is set pos2 in both corner of custom island to set and then /is set name custom island done when a player do /is create name it will create that custom island :D

**Comment by reveenge** [comment]
> > I don't understand what you call Custom Island, I would appreciate it if you could explain it again.
> 
> I mean we can able to make a custom island world and then we can do /is set pos1 and /is set pos2 in both corner of custom island to set and then /is set name custom island done when a player do /is create name it will create that custom island :D

it can be done but i am designing on a single island i can look at it later

**Comment by reveenge** [comment]
> > I will not remove the scoreboard system because I wrote it myself
> 
> Oh okay but can you add option to disable that in config and make skyblock x addon for ifera Scorehud 6.0.0 so less lag tags: {island_size} {island_lock_status} it shows island locked or not {island_members}?

I don't think so at the moment, you can add it yourself by changing the codes
